examples/workflows/use_cases/parquet_pandas.py

```diff
@@ -0,0 +1,92 @@
+"""This example shows how to use the custom serialisation features for third party libraries like Pandas. It also shows
+how we can easily use big file formats like parquet through Hera Artifact annotations.
+
+!!! note
+    You will require a "runner" image for this example, which requires the dependencies `pandas` and `pyarrow` to run.
```

Comment:
maybe this might be clearer?
```suggestion
    This example requires `pandas` and `pyarrow` to run. You will need to provide a "runner" image containing these dependencies.
```